package.json

```diff
@@ -247,6 +252,12 @@
         "category": "ONE"
       }
     ],
+    "keybindings": [
+      {
+          "command": "one.explorer.runSingleSelectedCfg",
+          "key": "ctrl+f5"
+      }
```

Comment:
I recommend adding 'when' condition here. 
Current way does enable your "runCfg" shortcut everywhere. 
You can make it only enabled when 
(1) One Explorer View is active.
(2) It has single selected cfg.

like
"when" : "OneExplorerView.active && one.explorer:hasSingleSelectedCfg"

'OneExplorerView.active' is a context value provided by vscode by default.
We can defined our own context one.explorer:hasSingleSelectedCfg here.

Reply:
Ok.

Reply:
You can find the example here: 
https://github.com/dayo09/ONE-vscode/tree/0203-review-1489

I checked it works well :-D

Reply:
Thanks, it works well :-D

Reply:
My pleasure to be useful )